pkgs/development/compilers/llvm/common/mlir/default.nix

```diff
@@ -17,22 +17,28 @@ stdenv.mkDerivation rec {
   src = runCommand "${pname}-src-${version}" {} ''
```

Comment:
I'd prefer to keep `pname` here, but let me know if it should be replaced by `mlir`.